quad_simulator/gazebo_scripts/package.xml

```diff
@@ -71,6 +71,13 @@
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>joint_state_controller</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
+
+  <depend>gazebo-ros-pkgs</depend>
+  <depend>ros-control</depend>
+  <depend>gazebo-ros-control</depend>
+  <depend>effort-controllers</depend>
+  <depend>imu-tools</depend>
+  <depend>message-to-tf</depend>
```

Comment:
Can't resolve message-to-tf rsodep key, CircleCI output below. Need to fix

ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
gazebo_scripts: Cannot locate rosdep definition for [message-to-tf]
a1_description: Cannot locate rosdep definition for [message-to-tf]
spirit_description: Cannot locate rosdep definition for [message-to-tf]
Continuing to install resolvable dependencies...

Reply:
Those are likely supposed to be underscores not hyphens, as they are referring to cmake package names rather than debian packages. Regardless, I'm not sure if that package is even being used, may be worth checking if things run fine without it?

Reply:
Any updates on this?

Reply:
It seems like everything works fine without the message_to_tf package. I'm not sure why we can't resolve the package, the rule for it [is here](https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml). Could someone else double check that things run fine without it, and then we can just remove it.